README.md

```diff
@@ -25,7 +25,7 @@ expect that the workflows will work just as well using other runners.
 The easiest way to get started is to install [bioconda][bioconda] and the run the following commands:
 
 ```bash
-conda create -n workflows-dev -c conda-forge cromwell==47 -y
+conda create -n workflows-dev -c conda-forge cromwell miniwdl -y
```

Comment:
Is there a reason to recommend installing both cromwell and miniwdl?

Reply:
It's more of a suggestion. I find if you're doing any development you want to be able to run `miniwdl check`.

Reply:
Personally, I wouldn't include it since we don't reference `miniwdl` anywhere else in the repository. But I'm fine to leave it in if we're planning on adding documentation around using `miniwdl` to check workflows.